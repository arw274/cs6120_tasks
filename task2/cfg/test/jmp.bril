# 0 -> {2}, 1 -> {2}, 2 -> {}
@main {
    a: int = const 0;
    jmp .end;
    b: int = const 0;
.end:
    ret;
}