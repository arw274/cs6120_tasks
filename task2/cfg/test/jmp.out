Block 0: [{'dest': 'a', 'op': 'const', 'type': 'int', 'value': 0}, {'labels': ['end'], 'op': 'jmp'}]
Block 1: [{'dest': 'b', 'op': 'const', 'type': 'int', 'value': 0}]
Block 2: [{'label': 'end'}, {'op': 'ret'}]
{'main': {0: [2], 1: [2], 2: []}}
